content-src/experiments/tracking-protection.yaml

```diff
@@ -91,6 +91,113 @@ created: '2016-09-22T00:07:28.847430Z'
 completed: '2017-02-15T14:00:00.000000Z'
 uninstalled: '2017-02-15T14:00:00.000000Z'
 eol_warning: We will automatically disable the Tracking Protection experiment and report the results when it ends.
+graduation_report: >
+  <h3>Test Pilot Tracking Protection Retirement Report</h3>
+  <p>The Tracking Protection feature in Private Browsing Mode is popular with Firefox users, 
+  but it comes with a few downsides: it can break page layout and multimedia content on 
+  websites, and it sometimes blocks content that isn’t tracking you. We’ve collected qualitative 
+  evidence of this through user feedback, but since we introduced Tracking Protection, 
+  we haven’t been able to quantify the impact of breakage and false positives on the overall 
+  user experience.</p>
+  <p>We launched the Tracking Protection experiment in Test Pilot with a goal of understanding
+   how the block list we use for Tracking Protection in <a target="_blank" href="https://support.mozilla.org/en-US/kb/private-browsing-use-firefox-without-history">Private Browsing Mode</a> and in 
```

Comment:
You could remove the /en-US/ in the address and let MDN redirect to the user's most appropriate locale.